set.mm

```diff
@@ -27622,13 +27628,13 @@ generally appear in a single form (either definitional, but more often
        Carneiro, 11-Aug-2016.)  (Proof modification is discouraged.)
        (New usage is discouraged.) $)
     nfcriiOLD $p |- ( y e. A -> A. x y e. A ) $=
-      ( vz cv wcel wsb wal nfcriv nfsbv nf5ri clelsb3 albii 3imtr3i ) EFCGZEBHZ
-      QAIBFCGZRAIQAPEBAAECDJKLEBCMZQRASNO $.
+      ( vz cv wcel wsb wal nfcri nfsbv nf5ri clelsb3 albii 3imtr3i ) EFCGZEBHZQ
+      AIBFCGZRAIQAPEBAAECDJKLEBCMZQRASNO $.
 
     $( Obsolete version of ~ nfcri as of 23-May-2024.  (Contributed by Mario
        Carneiro, 11-Aug-2016.)  (Proof modification is discouraged.)
        (New usage is discouraged.) $)
-    nfcriOLDOLD $p |- F/ x y e. A $=
+    nfcriOLDOLDOLD $p |- F/ x y e. A $=
```

Comment:
do we really need so many OLD versions? Since all modifications were done in a short time, the "intermediate" versions may not be necessary.

Reply:
He follows our usual pattern, which is what we want in 99% of the cases.  Here, I agree, there is an overkill of OLD variants with little extra insight into what we want to achieve.  Maybe we should simply squash the variants in an extra janitorial move outside of the user supplied contribution.  It should not block merging, in my eyes.